src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: "
+                + getArea() + " sq. units, firstLeg: "
+                + this.firstLeg + ", secondLeg: "
+                + this.secondLeg + " units, color: "
+                + this.color);
```

Comment:
Consider using `getColor()` instead of directly accessing the `color` field to maintain encapsulation and adhere to OOP principles.